src/nft/components/card/index.tsx

```diff
@@ -71,7 +71,9 @@ export const NftCard = ({
   const tokenType = collectionNft ? asset.tokenType : profileNft ? asset.asset_contract.tokenType : undefined
   const marketplace = collectionNft ? asset.marketplace : undefined
   const listedPrice =
-    profileNft && !isDisabled && asset.floor_sell_order_price ? floorFormatter(asset.floor_sell_order_price) : undefined
+    profileNft && !isDisabled && !hideDetails && asset.floor_sell_order_price
```

Comment:
this feels like mixing the concern of what the price is vs whether to show the price in a specific context. semantically i would prefer to make it an option on the tile component to display the price. 

you could imagine a case in the future where we want to give some special treatment to a tile based on a price threshold or something, and then that would require refactoring this without the benefit of design intent + context

Reply:
makes sense